deploy/helm/sumologic/requirements.yaml

```diff
@@ -8,9 +8,8 @@ dependencies:
     repository: https://kubernetes-charts.storage.googleapis.com/
     condition: prometheus-operator.enabled
   - name: falco
-    version: 1.1.7
-    repository: https://kubernetes-charts.storage.googleapis.com/
-    condition: falco.enabled
```

Comment:
the condition flag won't work with this repo?

Reply:
It will.  These conditions flags don't actually exist in the sub-charts.  It just matters that they exist in our values.yaml

Reply:
Ohh i see what you mean now, that should not have been removed...good catch. Fixed